core/build.gradle

```diff
@@ -1,17 +1,23 @@
-ext {
-    shadowJarEnabled = true
+plugins {
+    id "io.micronaut.build.internal.convention-core-library"
+}
+
+micronautBuild {
+    core {
+        documented = false
+    }
 }
 
 tasks.named("shadowJar") {
     exclude "module-info.class"
 }
 
 dependencies {
-    api libs.managed.slf4j
```

Comment:
are we removing slf4j from core? would not this be a breaking change?

Reply:
That library is added automatically by applying the `base convention plugin (transitively)